packages/strapi-admin/ee/admin/containers/Roles/CreatePage/ContentTypeCollapse/CollapsePropertyMatrix/utils/generateHeadersFromActions.js

```diff
@@ -0,0 +1,12 @@
+const generateHeadersFromActions = (actions, propertyName) => {
+  return actions.map(action => {
+    const isActionRelatedToCurrentProperty =
+      Array.isArray(action.applyToProperties) &&
+      action.applyToProperties.indexOf(propertyName) !== -1 &&
+      action.isDisplayed;
+
+    return { label: action.label, isActionRelatedToCurrentProperty };
```

Comment:
ℹ️ I like this kind of functions, it's easy to figure out what is a "Header for actions", I know that it has the following shape: 

`{label: string, isActionRelatedToCurrentProperty: boolean}`

👍🏻